.github/workflows/test-daap.yml

```
@@ -0,0 +1,39 @@
+---
+name: Test application code
+
+on: # yamllint disable-line rule:truthy
+  pull_request:
+    branches:
+      - main
+    paths:
+      - containers/**
+  push:
+    paths:
+      - containers/**
```

```suggestion
    paths:
      - containers/daap-*/**
  push:
    paths:
      - containers/daap-*/**
```

There are none DaaP containers